src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --primary-font-color: #000;
+  --secondary-font-color: #00acdc;
+  --primary-bg-color: #fff;
+  --secondary-bg-color: #e9e9e9;
+}
```

Comment:
🔥

Reply:
Thank you for review! It was simple but interesting and useful task.